content/en/news/releases/1.19.x/announcing-1.19/change-notes/index.md

```diff
@@ -0,0 +1,302 @@
+
+- **Removed** the `CNI_ENABLE_INSTALL`, `CNI_ENABLE_REINSTALL`, `SKIP_CNI_BINARIES`, and `UPDATE_CNI_BINARIES` feature flags.
+
+- **Removed** the support for deprecated envoy filter names in Envoy API name matches. Envoy filter will only be matched with canonical naming standard. See the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest/version_history/v1.14.0#deprecated) for more details.
+
+- **Removed** the `ISTIO_DEFAULT_REQUEST_TIMEOUT` feature flag. Please use timeout in VirtualService API.
+
+- **Removed** the `ENABLE_AUTO_MTLS_CHECK_POLICIES` feature flag.
+
+- **Removed** the `PILOT_ENABLE_LEGACY_AUTO_PASSTHROUGH` feature flag.
+
+- **Removed** the `PILOT_ENABLE_LEGACY_ISTIO_MUTUAL_CREDENTIAL_NAME` feature flag.
+
+- **Removed** the `PILOT_LEGACY_INGRESS_BEHAVIOR` feature flag.
+
+- **Removed** the `PILOT_ENABLE_ISTIO_TAGS` feature flag.
+
+- **Removed** the `ENABLE_LEGACY_LB_ALGORITHM_DEFAULT` feature flag.
+
+- **Removed** the `PILOT_PARTIAL_FULL_PUSHES` feature flag.
+
+- **Removed** the `PILOT_INBOUND_PROTOCOL_DETECTION_TIMEOUT` feature flag. This can be configured in MeshConfig if needed.
+
+- **Removed** the `AUTO_RELOAD_PLUGIN_CERTS` feature flag.
+
+- **Removed** the `PRIORITIZED_LEADER_ELECTION` feature flag.
+
+- **Removed** the `SIDECAR_IGNORE_PORT_IN_HOST_MATCH` feature flag.
+
+- **Removed** the `REWRITE_TCP_PROBES` feature flag.
+
+- **Removed** support for XDS v2 types in `EnvoyFilter`s. These should use the v3 interface. This has been a warning for multiple releases and is now upgraded to an error.
+
+- **Removed** the `PILOT_ENABLE_PROTOCOL_SNIFFING_FOR_OUTBOUND` and `PILOT_ENABLE_PROTOCOL_SNIFFING_FOR_INBOUND` feature flags. These have been enabled by default since Istio 1.5.
+
+- **Removed** support for looking up Envoy extensions in `EnvoyFilter` configuration by name without the typed config URL.
+
+- **Optimized** EnvoyFilter index generation to avoid rebuilding all `EnvoyFilters` every time a single one has changed, instead only rebuild the changed `EnvoyFilter` and update it in place.
+
+## Security
+
+- **Added** `insecureSkipVerify` implementation from DestinationRule. Setting `insecureSkipVerify` to `true` will disable CA certificate and Subject Alternative Names verification for the host.
+  ([Issue #33472](https://github.com/istio/istio/issues/33472))
+
+- **Added** support for PeerAuthentication policies in Ambient
+  ([Issue #42696](https://github.com/istio/istio/issues/42696))
+
+- **Added** `cipher_suites` support for non `ISTIO_MUTUAL` traffic through MeshConfig API.
+  ([Issue #28996](https://github.com/istio/istio/issues/28996))
+
+- **Added** Certificate Revocation List(CRL) support for peer certificate validation.
+
+- **Added** support for a flag called `USE_EXTERNAL_WORKLOAD_SDS`, when set to true, it will require an external SDS workload socket and it will prevent the istio-proxy from starting if the workload SDS socket is not found.
```

Comment:
```suggestion
- **Added** support for a flag called `USE_EXTERNAL_WORKLOAD_SDS`. When set to true, it will require an external SDS workload socket and it will prevent the istio-proxy from starting if the workload SDS socket is not found.
```